lib/zendesk_apps_tools/common.rb

```diff
@@ -20,17 +55,18 @@ def get_value_from_stdin(prompt, opts = {})
 
       while input = ask(prompt, thor_options)
         return '' if options[:allow_empty] && input.empty?
-        if input.to_s =~ options[:valid_regex]
-          break
-        else
-          say_error options[:error_msg]
-        end
+        break if input.to_s =~ options[:valid_regex]
+        say_error options[:error_msg]
       end
 
       input
     end
 
     def get_password_from_stdin(prompt)
+      if options[:unattended]
```

Comment:
good catch, this is included in a module that won't have options. Needs a failing test too.